src/core/client/stream/tabs/Comments/Comment/InReplyTo.tsx

```diff
@@ -2,21 +2,51 @@ import { Localized } from "@fluent/react/compat";
 import cn from "classnames";
 import React, { FunctionComponent } from "react";
 
+import { useCoralContext } from "coral-framework/lib/bootstrap";
+import { globalErrorReporter } from "coral-framework/lib/errors";
 import CLASSES from "coral-stream/classes";
+import computeCommentElementID from "coral-stream/tabs/Comments/Comment/computeCommentElementID";
 import { Flex, Icon } from "coral-ui/components/v2";
 
+import { CommentContainer_comment as CommentData } from "coral-stream/__generated__/CommentContainer_comment.graphql";
+
 import styles from "./InReplyTo.css";
 
 interface Props {
-  username: string;
+  parent: CommentData["parent"];
+  isLink: boolean;
 }
 
-const InReplyTo: FunctionComponent<Props> = ({ username }) => {
-  const Username = () => (
-    <span className={cn(styles.username, CLASSES.comment.inReplyTo.username)}>
-      {username}
-    </span>
-  );
+const InReplyTo: FunctionComponent<Props> = ({ parent, isLink }) => {
+  const { pym } = useCoralContext();
+
+  const navigateToParent = (id: string) => {
+    const elemId = computeCommentElementID(id);
+    const elem = document.getElementById(elemId);
+    if (elem) {
+      void pym?.scrollParentToChildEl(elemId);
+      elem.focus();
+    } else {
+      globalErrorReporter.report(
+        `Assertion Error: Expected to find parent comment with id ${parent?.id} but could not`
+      );
+    }
+  };
```

Comment:
Should use `useCallback` here and it also seems unnecessary to pass in the `id` because you already got this from `parent.id`.